---
permalink: /research/
title: "Research"
excerpt: ""
author_profile: true

---

Major projects
----
* **Mapping morphological malformation to genetic dysfunction in blood vessel organoids with 22q11.2 deletion syndrome (Finished)** 

    The 22q11.2 deletion syndrome (22q11.2 DS) is a genetic disorder caused by microdeletions in the chromosome 22. The deficits in blood vessel development, such as abnormal aortic arch morphology, tortuous retinal vessels, and kidney anomalies have been observed in the patients with 22q11.2 DS. However, how dysfunction endothelial cells and pericytes contribute to the vasculopathy remains unknown. In this study, we used the 3D self-assembled human blood vessel organoids (VOs) generated from 22q11.2 patients iPSCs to model the vascular malformation and genetic dysfunctions. Combined high resolution light-sheet 3D imaging,  single cell transcriptome analysis and integrating deep learning approach, we determined that the VOs with 22q11.2 deletion have smaller VO size. The alterations in 22q11.2DS VO size is the results of  instability in vasculature network formation while increased angiogenesis/sprouting. Voronoi diagram-based tessellation analysis also indicated the colocalization of endothelial tubes and mural cells was less than normal indicating that alterations in EC and mural interactions might also contribute to the deficits in vascular network formation. Single-cell transcriptome profiling revealed the variations of VO heterogeneity with 22q11.2 deletion, including the organ specificity of endothelial cells, and vascular developmental processes. Intercellular communications indicated the vascular dysfunction in 22q11.2 deletion was assocated with less cell-cell communications, and upregulation of ECM receptor in EC and vascular associated cells. We conclude that our study reveal important insight on pathogenesis of 22q11.2 deletion vasculopathy. 

* **Studying the intercellular interaction between stem cells and cancer cells (Ongoing)**

    Certain types of the stem cells, such as mesenchymal stem cell (MSC), neural stem cell (NSC) harbored the ability of tumor-homing, thus considered as an intelligent drug carrier for treating the cancer such as GBM. However, the mechanism of how these stem cells is regulated by the cancer cell signals, and how the trajectory of stem cells gets affected by their interaction with cancer cells is not well understood yet, which hinder the exploration of the more effective delivery strategy. Here we utilized the application of integrated deep learning methods to reveal the interaction dynamics of single and collective MSC/cancer motility. We quantified the cellular dynamics in the co-culturing system of the MSC and cancer cell. We model the co-culture system as an active nematic liquid crystal identified the cancer cell apoptosis induced by the formation of topological defects among the population of rod-shaped MSC. We modeled the interaction pattern of the co-culture by statistical generative model. Single cell RNA profiling revealed the affected signaling among the stem-cancer cell interaction. Our study provided a comprehensive physical insight for the stem cell and cancer cells interactive motility with novel analytical ways and provided potential suggestions for the delivery strategies. 

* **Deep learning models for integration histological images and spatial transcriptomics (Ongoing)**

    Collaborated with Azizi Lab
    
    Current technologies in spatial transcriptomics profiling are able to provide the tissue information that links to the genetic expression. However, the histological images have not yet been investigated to integrate with gene expression yet, which diminishes the values of the spatial profiling. Here, we develop comprehensive integration between histological images and the spatial transcriptomics to predict the locally variable features. 
    
    The datasets we analyzed include midbrain organoids (our own datasets), human dorsolateral pre-frontal cortex (published datasets) and breat cancer (published & our own datsets).  

* **Modeling radiation injury on blood vessel organoids (Ongoing)** 
    
    Collaborated with Vunjak-Novakovic Lab


* **Whole mouse brain vasculature with 22q11.2DS (Ongoing)**
   
   Collaborated with Tomer Lab

Other collaborated projects
----
* Emulating Early Atherosclerosis in a Vascular Microphysiological System Using Branched Tissue‐Engineered Blood Vessels (with Jounghyun Lee)
    * Simulation of blood flow in branched TEBV with openfoam/comsol
    
* CRISPR/Cas9‐mediated mutagenesis to validate the synergy between PARP1 inhibition and chemotherapy in BRCA1‐mutated breast cancer cells (with Yeh-Sing Lao)

* Supervised Tumor Cell Subtype Identification via SCAN (with Russell Kunes, class project)

* Towards better understanding of developmental disorders from integration of spatial single-cell transcriptomics and epigenomics (class project)

* Chronic opioid treatment arrests neurodevelopment and alters synaptic activity in human midbrain organoids (with Yang Xiao)

* Machine learning mediated quantification of breast cancer metastasis (with Tolu)

* some scRNAseq analysis (with Gordana Lab)

Experimental skills
----


Computational skills
----





